src/libmongoc/tests/unified/util.c

```diff
@@ -163,3 +163,31 @@ test_bson_util_install (TestSuite *suite)
    TestSuite_Add (
       suite, "/unified/selftest/util/copy_and_sort", test_copy_and_sort);
 }
+
+/* TODO (CDRIVER-3525) add test support for CMAP events once the C driver
+ * supports CMAP. */
+bool is_unsupported_event_type (const char* event_type) {
+   char * unsupported_event_types[] = {
```

Comment:
Is `char * unsupported_event_types` something `clang-format` suggests? I don't recall seeing isolated asterisks in any previous code. Likewise for `char ** iter` below.

Reply:
I forgot to clang-format. Updated.